### ISSUE TITLE
[Research] OUI Compliance audit of the `vis_type_table` plugin

### ISSUE THREAD

**Issue by BSFishy** [opensearch-project/OpenSearch-Dashboards#4099]
Tracking issue for OUI compliance audit of the `vis_type_table` plugin.

Follow the steps in #3945 to complete the audit, then open another issue with findings. Once the work of performing the audit is done, this issue can be closed.

<details>
  <summary>Related files</summary>

  size (bytes) | file | notes
---|---|---
584 | [src/plugins/vis_type_table/public/components/table_vis_app.scss](https://github.com/opensearch-project/OpenSearch-Dashboards/tree/main/src/plugins/vis_type_table/public/components/table_vis_app.scss) | 

</details>

If this issue isn't already assigned to someone and you'd like to take on this work, please ping @BSFishy in the comments and I can assign it to you.

**Comment by curq** [comment]
@BSFishy I can take this issue.

**Comment by curq** [comment]
@BSFishy Finished. https://github.com/opensearch-project/OpenSearch-Dashboards/issues/4099
